src/ert/run_models/base_run_model.py

```diff
@@ -109,6 +110,12 @@ def __init__(
             _seed_sequence(simulation_arguments.random_seed)
         )
         self.substitution_list = ert.ert_config.substitution_list
+        self.run_paths = Runpaths(
+            jobname_format=ert.ert_config.model_config.jobname_format_string,
+            runpath_format=ert.ert_config.model_config.runpath_format_string,
+            filename=str(ert.ert_config.runpath_file),
+            substitute=self.substitution_list.substitute_real_iter,
```

Comment:
This looks a little weird to me, and it would not be obvious to me that I need to pass `self.substitution_list.substitute_real_iter` to `substitute` here if I were changing this. Any context to what `substitute_real_iter` is @eivindjahren?

Reply:
It does substitution in the context of one realization and one iteration where we have the extra magicstrings `<IENS>` and `<ITER>`. Could probably rename it to `substitute_in_job` or `substitute_in_run`.